recipes/Diamond/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd src
+./install-boost &> /dev/null
```

Comment:
hah, the compiling instruction is different in the readme. I'll try cmake

On Mon, Nov 9, 2015 at 1:07 PM, Jorge Cañardo Alastuey <
notifications@github.com> wrote:

> In recipes/Diamond/build.sh
> <https://github.com/biocore/conda-recipes/pull/35#discussion_r44331579>:
>
> > @@ -0,0 +1,7 @@
> > +#!/bin/bash
> > +
> > +cd src
> > +./install-boost &> /dev/null
>
> Does boost actually end up being packaged in the binary? From looking at
> that script, it doesn't seem to be copied to $PREFIX.
>
> However, the cmake build instructions do seem to do that well, so maybe
> they should be used?
> https://github.com/bbuchfink/diamond#compiling-using-cmake
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/conda-recipes/pull/35/files#r44331579>.
>